src/gcl/MeshVelocityAlg.C

```diff
@@ -79,9 +79,12 @@ MeshVelocityAlg<AlgTraits>::MeshVelocityAlg(Realm& realm, stk::mesh::Part* part)
     NUM_IP, isoParCoords_, isoCoordsShapeFcnHostView_.data());
   Kokkos::deep_copy(isoCoordsShapeFcnDeviceView_, isoCoordsShapeFcnHostView_);
 
-  Kokkos::View<
-    const int**, Kokkos::HostSpace, Kokkos::MemoryTraits<Kokkos::Unmanaged>>
-    scsFaceNodeMapHostView(&scsFaceNodeMap_[0][0], 12, 4);
+  Kokkos::View<int[12][4], Kokkos::HostSpace> scsFaceNodeMapHostView;
```

Comment:
If we create this view using `Kokkos::create_mirror`, it will ensure that the layouts between host and device views that we're copying between are the same.  
```
  auto scsFaceNodeMapHostView = Kokkos::create_mirror(scsFaceNodeMapDeviceView_);
```